node/pkg/driver/device_connectivity/errors.go

```diff
@@ -21,6 +30,22 @@ func (e *MultipathDeviceNotFoundForVolumeError) Error() string {
 	return fmt.Sprintf("Couldn't find multipath device for volumeID [%s]. Please check the host connectivity to the storage.", e.VolumeId)
 }
 
+type MultipathDeviceNotFoundForVolumePathError struct {
+	VolumePath string
+}
+
+func (e *MultipathDeviceNotFoundForVolumePathError) Error() string {
+	return fmt.Sprintf("Couldn't find multipath device for VolumePath [%s]. Please check the host connectivity to the storage.", e.VolumePath)
```

Comment:
I'm not sure the host connectivity is the best suggestion here.
this scenario is probably similar to [this one](https://github.com/IBM/ibm-block-csi-driver/blob/v1.9.0/node/pkg/driver/node.go#L445).
maybe:
```suggestion
	return fmt.Sprintf("Couldn't find multipath device for VolumePath [%s]. Please verify the path is mounted", e.VolumePath)
```